bin/alluxio-start.sh

```diff
@@ -199,50 +242,34 @@ ensure_dirs
 
 case "${WHAT}" in
   all)
-    check_mount_mode $2
     if [[ "${killonstart}" != "no" ]]; then
       stop ${BIN}
     fi
-    start_master $3
+    start_master ${FORMAT}
     sleep 2
-    ${LAUNCHER} ${BIN}/alluxio-workers.sh ${BIN}/alluxio-start.sh worker $2
+
+    ${LAUNCHER} ${BIN}/alluxio-workers.sh ${BIN}/alluxio-start.sh worker ${MOPT}
```

Comment:
I am a little confused with the convection here.  I see lots of strings with variables unquoted in Alluxio bash scripts. Google style recommends quoting every string contains variable.  What is the convention that I should follow here?